product/src/main.js

```diff
@@ -1,20 +1,10 @@
-import express from 'express';
-import { router } from './routes.js';
 // const swaggerUi = require('swagger-ui-express');
 // const swaggerDocs = require('./swagger.json');
-import bodyParser from 'body-parser';
 import client from './repositories/databaseClient.js';
+import {app} from './app.js';
 
 const port = 3001;
```

Comment:
i think you can put the port on .env

Reply:
and you should create and .env.example so everyone knows what variables are needed to run